modules/diskq/tests/test_diskq_truncate.c

```diff
@@ -76,12 +76,12 @@ _save_diskqueue(LogQueue *q)
 }
 
 static gsize
-_calculate_serialized_empty_message_size(QDisk *qdisk)
+_calculate_serialized_empty_message_size(LogQueueDisk *queue_disk)
 {
   LogMessage *msg = log_msg_new_empty();
   GString *serialized_msg = g_string_new(NULL);
 
-  cr_assert(qdisk_serialize_msg(qdisk, msg, serialized_msg));
+  cr_assert(log_queue_disk_serialize_msg(queue_disk, msg, serialized_msg));
```

Comment:
This commit probably belongs to the first one, where `qdisk_serialize_msg()` is removed.
(We prefer creating commits that at least can be compiled.)